flink-runtime/src/test/java/org/apache/flink/runtime/io/network/buffer/LocalBufferPoolTest.java

```diff
@@ -306,7 +306,9 @@ public void testDestroyDuringBlockingRequest() throws Exception {
 
 				// Request all available buffers
 				for (int i = 0; i < numberOfBuffers; i++) {
-					requested.add(localBufferPool.requestBufferBlocking());
+					final Buffer buffer = localBufferPool.requestBuffer();
+					assertNotNull(buffer);
```

Comment:
nit: with `checkNotNull` you can inline the not null check with the assignments/method calls
```
buffer = checkNotNull(localBufferPool.requestBuffer());
```
or
```
requested.add(checkNotNull(localBufferPool.requestBuffer());
```